control_plane/roles/control_plane_customiso/files/temp_centos7.cfg

```diff
@@ -0,0 +1,56 @@
+# Install OS instead of upgrade
+install
+
+# SELinux configuration
+selinux --disabled
+
+# Firewall configuration
+firewall --disabled
+
+# text install
+text
+
+# Do not configure the X Window System
+skipx
+
+ignoredisk --only-use=sda
+
+# Keyboard layouts
+keyboard us
+
+# System language
+lang ks_language
+
+# Network information
+network  --bootproto=dhcp --device=ks_nic --onboot=on
+
+# Root password
+rootpw --iscrypted ks_password
+
+# System services
+services --enabled="chronyd"
+
+# System timezone
+timezone --utc ks_timezone
+
+# System bootloader configuration
+bootloader --location=mbr --boot-drive=sda
```

Comment:
I'm curious about defaulting to `/dev/sda` Will a BOSS card automatically be set as `/dev/sda`? Would we want to set an installed BOSS card as the default boot/install device?

@sujit-jadhav @j0hnL I'd appreciate feedback.